frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -223,6 +226,11 @@ export const funnelLogic = kea<funnelLogicType>({
                 return !!(featureFlags[FEATURE_FLAGS.FUNNEL_BAR_VIZ] && preflight?.is_clickhouse_enabled)
             },
         ],
+        funnelPersonsEnabled: [
+            () => [selectors.featureFlags, selectors.preflight],
+            (featureFlags, preflight) =>
+                featureFlags[FEATURE_FLAGS.FUNNEL_PERSONS_MODAL] && preflight?.is_clickhouse_enabled,
```

Comment:
check is added here now so it shouldn't disturb anything unrelated to this feature flag